superset-frontend/package.json

```diff
@@ -158,6 +158,7 @@
     "polished": "^3.7.2",
     "prop-types": "^15.7.2",
     "query-string": "^6.13.7",
+    "rc-resize-observer": "^1.2.0",
```

Comment:
Can you use `react-resize-detector` which is already a dependency?

Reply:
Great idea, happy to remove new dependency!

Reply:
Verified this works well and excited to NOT add a new dependency - thanks for the suggestion!  will add changes in next commit